### PR TITLE
feat(login): sign up feedback and google login

### PR DESCRIPTION
**Pull request by WilmerCo**
## Description
Adding an alert feedback after sign up and login success, also adding the login with google feature

## Can this PR be merged now?

 - [x] Yes, the task is completed and it is working as expected
 - [ ] No, this PR is a WIP and **MUST NOT BE MERGED YET** :)

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -26,11 +29,78 @@ export default function SignIn() {
     resolver: yupResolver(signInFormSchema)
   })
   const router = useRouter()
+  const alert = useAlert()
+  const clientId = '344445011201-knrbtue9u5o35rfndovhqjrj7m5627re.apps.googleusercontent.com'
```
Comment (Carlosarturo28) [inline src/pages/index.tsx:33]: Should we put this on an env file?

File: src/pages/signup.tsx [hunk 2]
```diff
@@ -33,13 +34,19 @@ export default function SignUp() {
...
 
   const handleSignUp: SubmitHandler<SignUpFormData> = async (values) => {
-    await HTTPHandler.post('/users', {
+    await HTTPHandler.post('users', {
       ...values
-    }).then(() => {
-      router.push('/')
     })
+      .then(() => {
+        alert.success('Sign up succesfull')
```
Comment (Carlosarturo28) [inline src/pages/signup.tsx:44]: typo: change succesfull to successful

**Review by Carlosarturo28 — COMMENTED** [review]
I'm not sure if we should use try/catch instead of then/catch in async/await.

File: src/pages/index.tsx (hunk 1 above)
Comment (WilmerCo) [reply src/pages/index.tsx:33]: you're right, thanks for the comment

**Pull request merged**